robocopkaka/event-manager-rails
# #163478992 Admins can edit centers (UI)

#### What does this PR do?
Give admins the ability to edit centres
#### Description of Task to be completed?
* Give admins the ability to edit centres
* Add a route for editing centres
* Create a component for editing a centre
#### How should this be manually tested?
* Visit `http//{baseURL}/centers/:id/edit`
* You should see a form pre-filled with the centre's details
* Edit the information and click submit
* You should see a toast message informing you that the centre was updated successfully
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories? (if applicable)
[#163478992](https://www.pivotaltracker.com/story/show/163478992)
